CONTRIBUTING.md

```diff
@@ -119,16 +119,23 @@ Before you submit your pull request consider the following guidelines:
 
 That's it! Thank you for your contribution!
 
-#### <a name="testing"></a> Testing approach
+#### <a name="testing"></a> Testing 
```

Comment:
Could you remove the whitespace after the anchor tag? It looks funny when rendered. 
Actually: what is that even for? Anchors are automatically generated by the GH markdown viewer for each heading...